{"failedTests":[{"testName":"parallel/test-without-async-context-frame","timestamp":1748123678352,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites Â» 0,win2019-COMPILED_BY-vs2022_clang #34438","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2019-COMPILED_BY-vs2022_clang/34438/","buildNumber":34438,"builtOn":"test-rackspace-win2019_vs2019-x64-3","nodeVersion":"25.0.0","commitHash":"ddd54022ab58a1f2d214fcc5e72d2aa6c9bdd747","callStack":[{"upstreamBuild":70632,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":80144,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":67046,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"58378"}],"tap":{"ok":false,"name":"parallel/test-without-async-context-frame","id":1025,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"flaky","exitcode":1,"stack":"Test failure: 'async-hooks\\test-async-local-storage-async-await.js'\nLocation: test\\parallel\\test-without-async-context-frame.mjs:49:5\nAssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-async-await.js failed with exit code 3221225477\n\n3221225477 !== 0\n\n    at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-without-async-context-frame.mjs:61:7)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Test.run (node:internal/test_runner/test:1069:7)\n    at async Suite.processPendingSubtests (node:internal/test_runner/test:752:7) {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 3221225477,\n  expected: 0,\n  operator: 'strictEqual'\n}"},"time":33018.288,"fullname":"parallel/test-without-async-context-frame","closingTestPoint":false}}]}